src/Storefront/Controller/ErrorController.php

```diff
@@ -63,7 +63,7 @@ public function error(\Throwable $exception, Request $request, SalesChannelConte
             } else {
                 $errorTemplate = $this->errorTemplateResolver->resolve($exception, $request);
 
-                if (!$request->isXmlHttpRequest()) {
+                if (!$request->isXmlHttpRequest() && !Feature::isActive('cache_rework')) {
```

Comment:
do i get it correctly that with 6.7 this whole if branch can be deleted? Can we just add a small comment about that?